local-interchaintest/src/utils/constants.rs

```diff
@@ -0,0 +1,36 @@
+pub const API_URL: &str = "http://127.0.0.1:42069";
+pub const WASM_EXTENSION: &str = "wasm";
+
+pub const NEUTRON_CHAIN: &str = "neutron";
+pub const NEUTRON_CHAIN_ID: &str = "localneutron-1";
+
+pub const GAIA_CHAIN: &str = "gaia";
+pub const GAIA_CHAIN_ID: &str = "localcosmos-1";
+
+pub const OSMOSIS_CHAIN: &str = "osmosis";
+pub const OSMOSIS_CHAIN_ID: &str = "localosmosis-1";
+
+pub const STRIDE_CHAIN: &str = "stride";
+pub const STRIDE_CHAIN_ID: &str = "localstride-3";
+
+pub const CHAIN_CONFIG_PATH: &str = "local-interchaintest/chains/neutron_gaia_osmosis_stride.json";
```

Comment:
let's add a note in the readme that tests should be executed from the root of the repo given how the paths are configured

Reply:
Added.